server/.env.example

```diff
@@ -9,9 +9,9 @@ REEARTH_ASSETBASEURL=https://localhost:8080/assets
 REEARTH_DEV=false
 
 # GCP
-GOOGLE_CLOUD_PROJECT=
-GCS_BUCKETNAME=
-GCS_PUBLICATIONCACHECONTROL=
+REEARTH_GOOGLE_CLOUD_PROJECT=
+REEARTH_GCS_BUCKETNAME=
+REEARTH_GCS_PUBLICATIONCACHECONTROL=
```

Comment:
Please make sure to edit envconfig from server/internal/app/config/config.go

Reply:
@pyshx 
thanks for comment.
I fixed.

`REEARTH_GOOGLE_CLOUD_PROJECT` => `GOOGLE_CLOUD_PROJECT`
Other is OK.